manubot/cite/handlers.py

```diff
@@ -419,207 +667,360 @@ def get_csl_item(self, citekey) -> Dict[str, Any]:
+    "icd11": "manubot.cite.curie.Handler_CURIE",
+    "icd9": "manubot.cite.curie.Handler_CURIE",
+    "icd9_2005": "manubot.cite.curie.Handler_CURIE",
+    "icd9cm": "manubot.cite.curie.Handler_CURIE",
+    "icd9cm_2005": "manubot.cite.curie.Handler_CURIE",
+    "icd9cm_2006": "manubot.cite.curie.Handler_CURIE",
     "icdc": "manubot.cite.curie.Handler_CURIE",
+    "icdcm_2005": "manubot.cite.curie.Handler_CURIE",
+    "icdo": "manubot.cite.curie.Handler_CURIE",
     "iceberg.element": "manubot.cite.curie.Handler_CURIE",
     "iceberg.family": "manubot.cite.curie.Handler_CURIE",
+    "iceo": "manubot.cite.curie.Handler_CURIE",
+    "ico": "manubot.cite.curie.Handler_CURIE",
     "ideal": "manubot.cite.curie.Handler_CURIE",
     "identifiers.namespace": "manubot.cite.curie.Handler_CURIE",
     "ido": "manubot.cite.curie.Handler_CURIE",
+    "idocovid19": "manubot.cite.curie.Handler_CURIE",
+    "idoden": "manubot.cite.curie.Handler_CURIE",
+    "idomal": "manubot.cite.curie.Handler_CURIE",
     "idoo": "manubot.cite.curie.Handler_CURIE",
     "idot": "manubot.cite.curie.Handler_CURIE",
     "idr": "manubot.cite.curie.Handler_CURIE",
     "iedb": "manubot.cite.curie.Handler_CURIE",
+    "iev": "manubot.cite.curie.Handler_CURIE",
     "igsn": "manubot.cite.curie.Handler_CURIE",
+    "ihw": "manubot.cite.curie.Handler_CURIE",
     "ilx": "manubot.cite.curie.Handler_CURIE",
+    "imdrf": "manubot.cite.curie.Handler_CURIE",
     "imex": "manubot.cite.curie.Handler_CURIE",
     "img.gene": "manubot.cite.curie.Handler_CURIE",
     "img.taxon": "manubot.cite.curie.Handler_CURIE",
     "imgt.hla": "manubot.cite.curie.Handler_CURIE",
     "imgt.ligm": "manubot.cite.curie.Handler_CURIE",
+    "imr": "manubot.cite.curie.Handler_CURIE",
     "inchi": "manubot.cite.curie.Handler_CURIE",
     "inchikey": "manubot.cite.curie.Handler_CURIE",
+    "inn": "manubot.cite.curie.Handler_CURIE",
+    "inn_id": "manubot.cite.curie.Handler_CURIE",
+    "ino": "manubot.cite.curie.Handler_CURIE",
     "insdc": "manubot.cite.curie.Handler_CURIE",
     "insdc.cds": "manubot.cite.curie.Handler_CURIE",
     "insdc.gca": "manubot.cite.curie.Handler_CURIE",
+    "insdc.run": "manubot.cite.curie.Handler_CURIE",
     "insdc.sra": "manubot.cite.curie.Handler_CURIE",
     "intact": "manubot.cite.curie.Handler_CURIE",
     "intact.molecule": "manubot.cite.curie.Handler_CURIE",
+    "intenz": "manubot.cite.curie.Handler_CURIE",
+    "interlex": "manubot.cite.curie.Handler_CURIE",
     "interpro": "manubot.cite.curie.Handler_CURIE",
+    "ip": "manubot.cite.curie.Handler_CURIE",
+    "ipd-imgt/hla": "manubot.cite.curie.Handler_CURIE",
+    "ipr": "manubot.cite.curie.Handler_CURIE",
     "ird.segment": "manubot.cite.curie.Handler_CURIE",
     "irefweb": "manubot.cite.curie.Handler_CURIE",
+    "iro": "manubot.cite.curie.Handler_CURIE",
     "isbn": "manubot.cite.isbn.Handler_ISBN",
+    "isbn-10": "manubot.cite.curie.Handler_CURIE",
+    "isbn-13": "manubot.cite.curie.Handler_CURIE",
```

Comment:
we might want to use `manubot.cite.isbn.Handler_ISBN` for `isbn-10` and `isbn-13`

Reply:
Yes, that makes sense to process all ISBNs consistently